creme/creme_config/forms/bricks.py

```diff
@@ -262,6 +267,14 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+LOCATIONS_MAP = {
+    BrickDetailviewLocation.TOP: 'top',
+    BrickDetailviewLocation.LEFT: 'left',
+    BrickDetailviewLocation.RIGHT: 'right',
+    BrickDetailviewLocation.BOTTOM: 'bottom',
+}
```

Comment:
Interesting problem of coupling & DRY. String "top", "left", ... are are coded in the JS code of the new widget ; so the could change the class attribute `zones = ('top', 'left', 'right', 'bottom')` in the widget (& the field), it would just not work.
I propose to 
- set 4 class constants" in our widget (`BOTTOM = "bottom"` etc.. plus a comment "# See JavaScript blabla..."); Maybe an Enum would be nice.
- the field uses the widget's constants/enum for its validation etc... (maybe it could define its own Enum which is a copy of the widget's one, or just refences it directly)
- Move LOCATIONS_MAP in _BrickDetailviewLocationsForm & use the above Enum values instead of hard-coded strings.